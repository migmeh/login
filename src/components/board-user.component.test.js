import React, { Component } from "react";
import { shallow } from 'enzyme';
import BoardUser from "./board-user.component";


describe('BoardUser component',()=>{

    it('BoardUser', ()=>{
        const wrapper = <BoardUser/>;
        expect(wrapper).toMatchSnapshot();
    });


    it('BoardUser shallow', ()=>{
        const wrapper = shallow(<BoardUser/>);
        expect(wrapper).toMatchSnapshot();
    });

});

