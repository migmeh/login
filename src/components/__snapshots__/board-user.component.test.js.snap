// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoardUser component BoardUser 1`] = `<BoardUser />`;

exports[`BoardUser component BoardUser shallow 1`] = `
<div
  className="container"
>
  <header
    className="jumbotron"
  >
    <h3 />
  </header>
</div>
`;
